### ISSUE TITLE
UI/UX design for Participating-org and Organisation-data Interface

### ISSUE THREAD

**Issue by anjesh** [younginnovations/aidstream-org-data#1]

**Comment by reidmporter** [comment]
(Mostly for @timgdavies) - possible/desirable to add some kind of "success rate" similar to how coupon code sites tell users "82% success rate" for a particular org-id.guide recommended list? Could be useful joint-advocacy tool for greater openness (rather than just individual single requests from users).

**Comment by timgdavies** [comment]
org-id.guide calculates both a relevance score (based on the query - changes for each list depending on what the search parameters were) and a quality score (based on the list meta-data, and constant for the list). 

These could be used in the display.

**Comment by anjesh** [comment]
Definitely. Currently we are just listing the org-id.guide list. The mockups can be seen [here](https://github.com/younginnovations/aidstream-org-data/issues/2#issuecomment-319874512)

**Comment by reidmporter** [comment]
Cross-posting to https://github.com/OpenDataServices/org-ids/issues/112, is it feasible to add a basic flag in addition to the score that functions as a "dead end" sign (for things like US EIN's which we know aren't public/open), vs. a "Come in, we're open!" sign (for things like UK Companies House which we know will result in joy, assuming you know the exact name of the company...because searching "Neon Tribe" won't return "neontribe" for some reason). This would be handy in both the org-id.guide UI and in AidStream's module.
